### ISSUE TITLE
Add `read_tsv()` and `write_tsv()` as aliases for `read_csv(separator='\t')` and `write_csv(separator='\t')`

### ISSUE THREAD

**Issue by Wainberg** [pola-rs/polars#10751]
### Problem description

TSVs tend to be encountered in the wild (much?) more often than CSVs, and I'm struck by how often I have to type `separator='\t'`. It's a source of bugs as well - I've already forgotten the `separator='\t'` a couple of times this week.

Would it be alright to add `read_tsv()` and `write_tsv()` as aliases for `read_csv(separator='\t')` and `write_csv(separator='\t')`?

**Comment by alexander-beedie** [comment]
No; tripling the API surface of a common function because one user comes across tabs a bit more often than commas is not good development practice - and it's unlikely that your experience is the same as everyone else's (anecdotally I _very_ rarely see TSV files, for example) ;)

**Comment by alexander-beedie** [comment]
No; tripling the API surface of a common function because one user comes across tabs a bit more often than commas is not good development practice - and it's unlikely that your experience is the same as everyone else's (anecdotally I _very_ rarely see TSV files, for example) ;)

**Comment by Shians** [comment]
I work in bioinformatics and TSV are very prevalent here. I would argue that having a function called `read_csv()` read anything that is not comma-separated value is a misleading interface, at the very least there should be a generic `read_table()` or `read_delim()`.
